analysis/analysis_utils.py

```diff
@@ -238,53 +238,60 @@ def save_results(self, res_filepath, out_suffix, keep):
             pickle.dump({"min_LH_per_px": keep["min_LH_per_px"], "num_res_per_px": keep["num_res_per_px"]}, f)
         np.save("%s/all_ps_%s.npy" % (res_filepath, out_suffix), stamps_to_save)
 
-    def _calc_ecliptic_angle(self, test_wcs, angle_to_ecliptic=0.0):
+    def _calc_ecliptic_angle(self, wcs, angle_to_ecliptic=0.0):
```

Comment:
Given that angle_to_ecliptic is no longer used, we can remove it from the function altogether.